### PR TITLE
ADD: New activation functions

### PR DESCRIPTION
**Pull request by sbharadwajj**
New activation functions added: SELU, ELU, LeakyReLU, LogSigmoid. 

Note: README-dev.md not updated in this pull request.

### CODE REVIEW

File: backpack/core/derivatives/elu.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        return False
+
+    def df(self, module, g_inp, g_out):
+        """First ELU derivative: `ELU'(x) = alpha * e^x if x < 0 else 1`. """
+        df_ELU = gt(module.input0, 0).float()
+        df_ELU[df_ELU == 0] = module.alpha * exp(module.input0[df_ELU == 0])
+        return df_ELU
+
+    def d2f(self, module, g_inp, g_out):
+        """Second ELU derivative: `ELU'(x) = alpha * e^x if x < 0 else 1`. """
```
Comment (f-dangel) [inline backpack/core/derivatives/elu.py:22]: Please change to ELU''

File: backpack/core/derivatives/selu.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+        df_SELU = gt(module.input0, 0).float()
+        df_SELU[df_SELU == 1] = self.scale
+        df_SELU[df_SELU == 0] = (
+            self.scale * self.alpha * exp(module.input0[df_SELU == 0])
+        )
+        return df_SELU
+
+    def d2f(self, module, g_inp, g_out):
+        """Second SELU derivative: `SELU'(x) = 0 if x < 0 else scale*alpha*e^x`. """
```
Comment (f-dangel) [inline backpack/core/derivatives/selu.py:31]: Change to SELU''

File: backpack/core/derivatives/elu.py (hunk 1 above)
Comment (sbharadwajj) [reply backpack/core/derivatives/elu.py]: changed

File: backpack/core/derivatives/selu.py (hunk 2 above)
Comment (sbharadwajj) [reply backpack/core/derivatives/selu.py]: changed

**Pull request merged**